extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesDeploy.java

```diff
@@ -68,8 +68,8 @@ private Result doCheck() {
         String masterURL = client.getConfiguration().getMasterUrl();
         try {
             masterURL = client.getConfiguration().getMasterUrl();
-            //Let's check id we can connect.
-            RootPaths paths = client.rootPaths();
+            //Let's check if we can connect.
+            client.getVersion();
             log.info("Kubernetes API Server at '" + masterURL + "' successfully contacted.");
```

Comment:
Show the version returned above here :wink:

Reply:
We could do that, but maybe it's a little to verbose?

Reply:
Maybe, perhaps it should be shown in a log.debug instead

Reply:
Done